### PR TITLE
tombstone: Mark `expiration_epoch` field as deprecated

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

File: tombstone/types.proto [hunk 1]
```diff
@@ -14,7 +14,8 @@ message Tombstone {
   // creator depending on the current NeoFS network settings. A tombstone object
   // must have the same expiration epoch value in `__NEOFS__EXPIRATION_EPOCH`
   // attribute. Otherwise, the tombstone will be rejected by a storage node.
```
Comment (roman-khimov) [inline tombstone/types.proto:16]: The behavior should be clarified here, either we don't care about it at all after deprecation, or it must be set in the way described.

Comment (cthulhu-rider) [reply tombstone/types.proto]: done

**Review by roman-khimov — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,10 +10,12 @@
 
 ### Changed
 - Split ID is deprecated, the first child part is used instead and is known to the other parts (#283)
+- `Tombstone.expiration_epoch` field is ignored by servers now
```
Comment (carpawell) [inline CHANGELOG.md:13]: no issue number

**Review by carpawell — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (cthulhu-rider) [reply CHANGELOG.md]: added

**Review by roman-khimov — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**